src/pages/FormBuilder/FormBuilder.module.css

```
@@ -367,51 +270,75 @@
 }
 
-.sectionDeleteBtn {
-  color: #DC3E3E;
-  border: 1.5px solid currentColor;
-  border-radius: var(--radius-sm);
-  box-sizing: border-box;
+.saveAndCancelBtns {
+  padding: 0 1.5rem;
+  min-width: max-content;
+  width: 6rem;
+  border: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)